flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/util/JobManagerWatermarkTracker.java

```diff
@@ -92,6 +104,7 @@ public long getUpdateTimeoutCount() {
     protected static class WatermarkUpdate implements Serializable {
         protected long watermark = Long.MIN_VALUE;
         protected String id;
+        protected boolean updateLocalWatermark = true;
```

Comment:
@sethsaperstein-lyft instead of adding this field and `getWatermark`, wouldn't it be simple to use `Long.MIN_VALUE` as a special value with `updateWatermark`? We already use `Long.MIN_VALUE` as "no watermark" in the fetcher and can designate is as idle in the accumulator.

Reply:
good suggestion. Thanks for taking a look. Updated.